src/screens/Home/Home.jsx

```diff
@@ -1,56 +1,62 @@
-import React, { useEffect, useState } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 
-import { getCountriesAction } from 'shared/store/countires/actions';
-import { getPlacesAction } from 'shared/store/places/actions';
+import {
+  getOriginPlaceAction,
+  getDestinationPlaceAction,
+} from 'shared/store/places/actions';
 import Button from 'shared/components/Button/Button';
 import SearchInput from 'shared/components/SearchInput/SearchInput';
 
 import './styles.scss';
 
 function Home() {
-  const [chosenCountry, setChosenCountry] = useState('');
-  const [chosenPlace, setChosenPlace] = useState('');
+  const [placeOrigin, setPlaceOrigin] = useState('');
+  const [placeDestination, setPlaceDestination] = useState('');
 
-  const { countries } = useSelector((state) => state.countries);
-  const isLoadingCountries = useSelector((state) => state.countries.loading);
-  const { places } = useSelector((state) => state.places);
-  const isLoadingPlaces = useSelector((state) => state.places.loading);
+  const { originPlaces } = useSelector((state) => state.places);
+  const { destinationPlaces } = useSelector((state) => state.places);
+  const { isLoadingOriginPlaces } = useSelector((state) => state.places);
+  const { isLoadingDestinationPlaces } = useSelector((state) => state.places);
 
   const dispatch = useDispatch();
 
-  useEffect(() => {
-    dispatch(getCountriesAction());
-  }, [dispatch]);
+  function getDestinationPlaces(value) {
+    dispatch(getDestinationPlaceAction(value));
+  }
 
-  useEffect(() => {
-    if (chosenCountry) {
-      dispatch(getPlacesAction(chosenCountry));
-    }
-  }, [dispatch, chosenCountry]);
+  function getOriginplaces(value) {
+    dispatch(getOriginPlaceAction(value));
+  }
 
   return (
     <div className="main">
       <div className="main__title">
         <h1 className="main__title-text">Let the journey begin</h1>
       </div>
-      <div className="main__search">
-        <SearchInput
-          items={countries}
-          setChosenItem={setChosenCountry}
-          inputName="country"
-          inputPlaceholder="Search countries"
-          isLoadingItems={isLoadingCountries}
-        />
-        {chosenCountry ? (
+      <div className="main__search main--column">
+        <div className="main__search-from">
+          <span>From</span>
           <SearchInput
-            items={places}
-            setChosenItem={setChosenPlace}
-            inputName="place"
-            inputPlaceholder="Search place"
-            isLoadingItems={isLoadingPlaces}
+            items={originPlaces}
+            setChosenItem={setPlaceOrigin}
+            inputName="origin"
+            inputPlaceholder="Where do You want to fly"
```

Comment:
Doesn't seem to be correct text for origin place.

Reply:
Fixed